packages/client/src/components/form/Redirect.tsx

```diff
@@ -8,34 +8,104 @@
  *
  * Copyright (C) The OpenCRVS Authors located at https://github.com/opencrvs/opencrvs-core/blob/master/AUTHORS.
  */
-import { IFormData, IFormSectionData } from '@client/forms'
+import {
+  IFormData,
+  IFormField,
+  IFormFieldValue,
+  IFormSectionData,
+  IHttpFormField,
+  Ii18nRedirectFormField
+} from '@client/forms'
 import { evalExpressionInFieldDefinition } from '@client/forms/utils'
 import { getOfflineData } from '@client/offline/selectors'
 import { getUserDetails } from '@client/profile/profileSelectors'
-import React from 'react'
+import React, { useEffect } from 'react'
 import { useSelector } from 'react-redux'
-import { Navigate } from 'react-router-dom'
+import { Link } from 'react-router-dom'
+import { useHttp } from './http'
+import { Button, getTheme, Icon } from '@opencrvs/components'
+import { useWindowSize } from '@opencrvs/components/src/hooks'
 
 export const RedirectField = ({
```

Comment:
Is this more of a Link Button rather than a Redirect nowadays?

Reply:
Yea, a secondary link button to align with the design. I believe there can be some good alternatives